openapi/generator_test.go

```diff
@@ -48,6 +48,7 @@ type (
 		*u
 		uu *u // ignored, unexported field
 		q     // ignored, embedded field of non-struct type
+		*Q `json:"data"`
```

Comment:
Could you please add another field that is an exported field of struct type, but that has no tag? To test that its fiels are then inlined.

Reply:
Also, could you keep the tag value to `Q` to match the current output, where each field is named by its type in the final spec.